### PR TITLE
backup: increase default snapshot timeout

### PR DESCRIPTION
**Pull request by immesys**
For larger databases, a snapshot can take much longer than the 5 seconds previously given. This approach is not exactly elegant, but for the snapshot in the backup, give it a larger timeout.

### CODE REVIEW

**Comment by xiang90** [comment]
lgtm.

do you mind updating the changelog?

**Comment by immesys** [comment]
Like that? I was not sure what format you were looking for

**Comment by xiang90** [comment]
that is what i am looking for.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Timeout for snapshots done by backup sidecar increased from 5 seconds to 1 minute
```
Comment (xiang90) [inline CHANGELOG.md]: Default timeout for snapshots ...

**Comment by xiang90** [comment]
lgtm

**Comment by hongchaodeng** [comment]
@immesys 
Can you address the comment?

**Comment by immesys** [comment]
@hongchaodeng sure. Done

**Comment by hongchaodeng** [comment]
Jenkins infra is upgrading. I'm merging since it is only doc change.

**Comment by hongchaodeng** [comment]
Thanks @immesys for the PR

**Comment by immesys** [comment]
No problem. Thanks for etcd-operator!

**Pull request merged**